### DISCUSSION TITLE
How are you using or want to use SymForce? Tell us here!

### DISCUSSION THREAD

**Discussion by hayk-skydio** [symforce-org/symforce#208] [General]
Drop in your use case for discussion :)

**Comment by sudo-robot-destroy** [comment]
At the moment I'm just using it to learn more about factor graphs. But I have plans to create a ROS package that uses it for a flexible plug and play factor graph system for state estimation.

**Comment by asa** [comment]
How about modeling a typical motion capture system? 
Typical setup would be  12 cameras on fixed mounts in space, pointing inward toward a robot with a fixed set of say 10 landmarks affixed to it. This seems to map to the fixed_size problem you have in examples already. I would love to see discussed how to extend this to be able to estimate velocities / twists in this kind of problem. I would also like to estimate the arrangement of these cameras in space in a calibration phase, then use these estimates to estimate the state of the robot driving around inside it. Presumably one could just run the system and do this all online.  What do you think?
